server/monitor_test.go

```diff
@@ -377,6 +377,46 @@ func TestMonitorServerz(t *testing.T) {
 	monitorExpectStatus(t, ServerPath, http.StatusInternalServerError)
 }
 
+func TestMonitorIsFTActiveStandaloneServer(t *testing.T) {
+	expectedStatusCode := http.StatusNoContent
+
+	resetPreviousHTTPConnections()
+	s := runMonitorServer(t, GetDefaultOptions())
+	defer s.Shutdown()
+
+	resp, _ := getBodyEx(t, http.DefaultClient, "http", IsFTActivePath, expectedStatusCode, "")
+	defer resp.Body.Close()
+
+	if expectedStatusCode != resp.StatusCode {
```

Comment:
You don't need this check since getBodyEx() will already ensure that the response status matched the one you pass as a parameter.